ddtrace/propagation/http.py

```diff
@@ -110,6 +110,8 @@ def _hex_id_to_dd_id(hex_id):
     If the id is > 64 bit then truncate the trailing 64 bit.
```

Comment:
```suggestion
    If the id is > 64 bit and 128bit trace_id generation is disabled then truncate the trailing 64 bit.
    Otherwise convert the full trace id into lower case hex values.
```